deploy/.env.example

```
@@ -0,0 +1,15 @@
+TS_AUTHKEY=tskey-auth-example
+# If you change TS_HOSTNAME, update deploy/tailscale/serve.json to use the
+# same hostname in its Web and AllowFunnel keys.
+TS_HOSTNAME=tailscale-mcp
+
+MCP_HTTP_BEARER_TOKEN=replace-with-at-least-32-random-characters
+MCP_HTTP_PORT=3000
+MCP_ALLOWED_HOSTS=tailscale-mcp,localhost,127.0.0.1
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The MCP_ALLOWED_HOSTS key should go before the MCP_HTTP_BEARER_TOKEN key

(UnorderedKey)